iOS Client/ArticlesController/MBArticleDetailViewController.swift

```diff
@@ -53,6 +58,23 @@ class MBArticleDetailViewController: UIViewController, WKNavigationDelegate {
         }
     }
     
+   @objc func bookmark(_ sender: Any) {
```

Comment:
is this @objc stuff necessary. Can't we do an IBAction? Idk about this, it's been a while